src/Service.Host/client/src/components/Root.js

```diff
@@ -107,6 +109,14 @@ function Root() {
                         path="/stores2/storage/locations/create"
                         element={<StorageLocation creating />}
                     />
+                    <Route
+                        path="/stores2/reports/daily/eu/dispatch"
+                        element={<DailyEuDispatchReport />}
+                    />
+                    <Route
+                        path="/stores2/reports/daily/eu/import/rsn"
+                        element={<DailyEuImportRsnReport />}
+                    />
```

Comment:
urls should be something like

stores2/reports/daily-eu-dispatch
and
/stores2/reports/daily-eu-import-rsn

so that every url segment has meaning, i.e.  it can be understood by client routing and makes sense for the API